src/components/Nav.jsx

```diff
@@ -7,11 +7,8 @@ import {
   StyledOcticon,
 } from "@primer/react";
 import { TableIcon } from "@primer/octicons-react";
-import useAuthState from "../hooks/use-auth-state";
-
-export default function Nav() {
-  const { authState, logout } = useAuthState();
```

Comment:
@gr2m just FYI: The usage of `useAuthState` hook was not working as expected here:
- When invoking `useAuthState()`, this creates a new React state associated to this component, but is not interconnected with an upper state of the application. ([see implementation](https://github.com/project-forms/project-forms.github.io/blob/main/src/hooks/use-auth-state.js))
- This means the local variable `authState` is always empty `{ user: {} }`. `setAuthState` for this local state is never invoked.